base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -208,6 +210,75 @@ static BOOL get_process_name_from_pid(DWORD pid, WCHAR *buf, DWORD chars)
  *
  * A PID of zero causes taskkill to warn about the inability to terminate
  * system processes. */
+
+
+static BOOL get_pid_creation_time(DWORD pid, FILETIME *time)
+{
+    HANDLE process = INVALID_HANDLE_VALUE;
+    FILETIME t1 = { 0 }, t2 = { 0 }, t3 = { 0 };
+    BOOL result = TRUE;
+
+    process = OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, pid);
+    if (!process)
+    {
+        result = FALSE;
+    }
+
+    if (!GetProcessTimes(process, time, &t1, &t2, &t3))
+    {
+        result = FALSE;
+    }
+
+    if (process != INVALID_HANDLE_VALUE || !process)
```

Comment:
If the suggestions from above have been correctly applied this would allow removing this `if`.

Reply:
Sorry for my bad, I fixed 😄